src/main/java/org/folio/inventory/consortium/model/ConsortiumEvenType.java

```diff
@@ -0,0 +1,24 @@
+package org.folio.inventory.consortium.model;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum ConsortiumEvenType {
```

Comment:
The name should reflect "instance" and "sharing". Unless you want to have an enum for all ECS events, then there is a typo - ConsortiumEventTypes

Reply:
Fixed